Gemfile

```diff
@@ -5,3 +5,7 @@ gem "command_line_reporter", '~> 3.0'
 gem "ruby-terminfo", '~> 0.1.1'
 gem "diffy", '~> 3.1'
 gem "hrx", '~> 1.0'
+
+group :development do
```

Comment:
I'm not sure it makes sense to have a separate "development" section, since this will only ever really be run in development mode.

Reply:
Totally fair!